cmd/otelcontribcol/go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.80.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/mezmoexporter v0.80.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.80.0
+  github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opensearchexporter v0.80.0
```

Comment:
Spacing on this line is off compared to the other lines

Reply:
Thanks!